OpenRA.Mods.Cnc/Traits/Render/WithCargo.cs

```diff
@@ -45,6 +45,16 @@ public WithCargo(Actor self, WithCargoInfo info)
 			body = self.Trait<IBodyOrientation>();
 		}
 
+		public void Tick(Actor self)
+		{
+			foreach (var c in cargo.Passengers)
+			{
+				var cargoFacing = c.TraitOrDefault<IFacing>();
+				if (facing != null && cargoFacing != null)
```

Comment:
Right. You can actually just skip the whole loop then. Mind to do that simple change so we can release?